Script to automatically generate badges from registration database (#2)
LSSTDESC/name-tag-guru: latex/main.tex

%% To include CJK character, one needs to use xelatex to compile
%% and also to have a CJK font installed (current set to `Noto Sans CJK TC`)
%% Otherwise, comment out the following two lines.
%\usepackage{xeCJK}
%\setCJKmainfont{Noto Sans CJK TC}

%% basic font settings
\usepackage[T1]{fontenc}
\usepackage[default]{raleway}
\providecommand{\raleway}{} % in case we are not using xeLaTex
\newcommand*{\latinfont}{\raleway}
\hyphenpenalty=10000

%% essential packages

> \usepackage{ticket} %when ready to print, remove the "boxed" option

Reviewer: ```suggestion
\usepackage{ticket} %to see the badge boundary (for testing purpose), add [boxed] as an option
```

Author: ooh yes, my bad, I removed that by accident. I think it might actually be better to keep the [boxed] by default

\renewcommand*{\ticketdefault}{%
  \put(10,2){\lbox{\includegraphics[width=1in]{desc-logo}}}%
  \put(380,14){\rbox{\includegraphics[width=0.7in]{lsstc-logo}}}%

>   \put(205,29){\cbox{\latinfont\scriptsize Collaboration Meeting, July.~23--27, 2018}}%

Reviewer: ```suggestion
  \put(205,29){\cbox{\latinfont\scriptsize Collaboration Meeting, July.~23--27, 2018}}% Update dates as needed
```

Author: yes :-D

>   \put(205,15){\cbox{\latinfont\scriptsize Carnegie Mellon University}}%

Reviewer: ```suggestion
  \put(205,15){\cbox{\latinfont\scriptsize Carnegie Mellon University}}% Update location as needed
```

Author: also yes :-D
